requirements/edx/coverage.in

```diff
@@ -13,5 +13,5 @@
 -c ../constraints.txt
 
 coverage                            # Code coverage testing for Python
-diff-cover                          # Automatically find diff lines that need test coverage
+diff-cover==4.2.3                   # Automatically find diff lines that need test coverage
```

Comment:
@asadmanzoor93 Why do we need to pin the `diff-cover` ?

Reply:
@zubair-arbi `diff-cover` also dropped support for `python3.5` in the latest release.